src/worker.py

```
@@ -1205,6 +1226,24 @@ async def api_create_session(req, env):
+        ).bind(act_id).all()
+        for row in enrolled.results or []:
+            if row.user_id != user["id"]:
+                await _create_notification(
+                    env,
+                    user_id=row.user_id,
+                    type_="new_session",
+                    title="New session scheduled",
+                    message=f"A new session '{title}' has been scheduled.",
+                    related_id=sid,
+                )
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)